best-practice-certificate-provisioning.md

```diff
@@ -188,7 +188,7 @@ The EST Server MAY also support manual authentication of the EST Client if:
 - The deployment chooses to enforce an extra authentication step.
 The exact process for manual authentication will be implementation specific, but the EST Server MUST provide enough information to the operator so they can authenticate the EST Client. During the manual authentication the EST Server MUST respond with either HTTP 202 or HTTP 503 and the response MUST include a `Retry-After` header.
 
-The EST Server MUST be capable of issuing TLS Certificates signed with both RSA and ECDSA keys.
+The EST Server MUST be capable of issuing TLS Certificates signed for all Digital Signature algorithms.
```

Comment:
Is 'all' defined anywhere? It's probably clear in the context of BCP-003-01, but perhaps not otherwise?